website/content/docs/overview/architecture/index.md

```diff
@@ -9,25 +9,57 @@ weight: 10
 The information below is outdated
 {{</ warning >}}
 
-{{< name >}} takes a **modular** approach to provide an open platform for controlling and securing the use of data across an organization. The figure below showcases the current architecture of the {{< name >}} platform, running on top of OpenShift Container Platform. The storage systems shown in the lower half of the figure are merely an example.
+{{< name >}} takes a **modular** approach to provide an open platform for controlling
+ and securing the use of data across an organization. The figure below showcases the
+  current architecture of the {{< name >}} platform, running on top of Kubernetes. 
+ The storage systems shown in the lower half of the figure are merely an example.
 
+The core parts of {{< name >}} are based on [Kubernetes operators](https://www.openshift.com/learn/topics/operators) 
+ and custom resource definition in order to define its work items and reconcile the state of
+ them.
 
-![Architecture](architecture.png)
+The primary interaction object for a data user is the `M4DApplication` CRD where a user defines which
+data should be used for which purpose. The following chart and description describe the architecture and components of
+{{< name >}} relative to when they are used.
 
-{{< bullet n=1 >}} A {{< name >}} operator configures the resources available to {{< name >}}: compute and storage resources used to run governance actions and optimize data access performace, and `M4DModule` resources that describe modules injectable into the data path.
+![Architecture](workflow_multicluster.png)
 
-{{< bullet n=2 >}} A data streward configures policies in an external policy manager over assets defined in an external data catalog. The policy manager and data catalog are connected to {{< name >}} `PolicyCompiler` and `DataCatalog` services via `PolicyManagerAdapter` and `DataCatalogAdapter`, respectively.
+Before the data user can perform any actions a data operator has to [install](../../setup/quickstart) the {{< name >}} and modules
+that can be used by the user. [Modules](../modules) are a way that {{< name >}} abstracts data access for
+users. Modules can provide read/write access or produce implicit copies that serve as lower latency caches of
+remote assets. Modules also enforce policies defined for data assets. A detailed description of them can be found [here](../modules).
 
-{{< bullet n=3 >}} A developer submits a `M4DApplication` resource (e.g., via CI/CD pipeline) holding metadata about the application. such metadata includes the data assets required by the application and the processing purpose.
+{{< name >}} connects to external services to retrieve information about policies, available data and credentials.
+Policies, assets and access credentials to the assets have to be defined before the user can create an application. The
+current abstraction knows 3 different connectors but depending on the implementation the underlying system
```

Comment:
Instead of "know" I would suggest "supports" and instead of "underlying system", "external system"